### DISCUSSION TITLE
osu!lazer Mod Customization Suggestions

### DISCUSSION THREAD

**Discussion by Cyrolia** [ppy/osu#16130] [Ideas]
# Sudden Death
**Minimum Combo** - Changes SD so that you only fail if you miss when your combo is below x (i.e. if you miss at 245 combo, and x is equal to 300, you would fail)

There would be a switch to change overall combo to max combo (i.e. if y = maximum combo achieved in the run, if you miss at 322 combo, but both x and y are equal to 500, you would not fail.)

**Mercy** - Allow for x amount of misses before fail

**Auto-Retry** - Adds the automatic retry feature of PF to SD

# Perfect

**Minimum Accuracy** - Similar to minimum combo for sudden death, but for accuracy instead of maximum combo (i.e. rather than failing immediately upon your accuracy dropping below 100, it could be set to fail when your accuracy drops below 95)

# Flashlight
**FOV Adjustment** - Self explanatory, changes how much you can see around your cursor

**Fov Change** - Changes the combo at which the flashlights FOV is reduced from 200 to a custom amount

would also have an option to remove the feature entirely

# Final Words 

Im not the best at putting my thoughts into words, so if you need further clarification on any of the ideas above please let me know.

**Edited for better readability**

Thanks!

-Cyro-

**Comment by mk56-spn** [comment]
these all seem fun and rather easy to implement in principal. ill tackle a few of these if im given the go ahead by someone from the osu team

**Reply by Cyrolia** [reply]
Great! Im glad my input could help ☺️

**Comment by Rekunan** [comment]
> Auto-Retry

If you check the customization of SD there's already a toggle for that (labeled "Retry on fail"), same with PF.

**Comment by Theighlin** [comment]
I think most of this and/or the general idea of requirements to pass a map fall into #13120 territory.

**Comment by mk56-spn** [comment]
@peppy is it ok if i make a PR for the changes suggested here for flashlight (modifiable radius) with this as a reference? or would you like a separate discussion, i should also note they dont overlap with the changes MBmasher made in #15180

**Reply by peppy** [reply]
i think it's fine to add it
